Movie_App.js

```diff
@@ -0,0 +1,128 @@
+document.addEventListener("DOMContentLoaded", () => {
+    fillSelectTag();
+    getIds();
+
+    window.addEventListener("load", start, false);
+
+    let form1 = document.querySelector('#form')
+    form1.addEventListener('submit', (event) => {
+        event.preventDefault();
+        let review = document.querySelector('#review');
+        // let newItem = document.querySelector('#new_item');
+
+        let reviewLocation = document.querySelector('#submited-reviews');
+        let newReview = document.createElement("li");
+
+        let bold = document.createElement("B");
+        let title = document.createTextNode(currentMovie());
+
+        let userReview = document.createTextNode(": " + review.value);
+        
+        bold.appendChild(title);
+
+        newReview.appendChild(bold);
+        newReview.appendChild(userReview);
+
+        reviewLocation.appendChild(newReview);
+        console.log("booyah")
+        //It took me forever to figure this out >.< Look at my pain!
+        // reviewLocation.appendChild(bold);
+        // boldWords.appendChild(bold)
+        // thisTitle.fontWeight = "bold";
+        // newReview.innerText = ": " + review.value;
+    })
+
+})
+
+function fillSelectTag () {
+    //I will pull the 20 movies from the api and put them in my select tag
+    fetch("https://ghibliapi.herokuapp.com/films/")
+    .then((response) => {
+        // console.log(response.json());
+        return response.json();
+    })
+    .then(movies => {
+        for (let movie of movies) {
+            let movieSelection = document.querySelector('#select-movie');
+            let newMovie = document.createElement("option");
+
+            newMovie.innerText = movie.title;
+
+            movieSelection.appendChild(newMovie);
+        }
+    })
+}
+
+function start() {
+    document.querySelector("#select-movie").addEventListener("change", getIds, false);
+}
+
+function currentMovie () {
+    return document.querySelector('#select-movie').value;
+}
+
+
+function getIds () {
```

Comment:
Definitely an interesting way of getting movie data, but here is another way that can save number of requests from the API. Since films endpoint already contains all movie data, we can also save the fetch result from a single request, then re-use that data by saving it in an array or as an object. It is not a big deal for a small app but reducing number of API requests will become important later on.